Offices/Offices.API/Program.cs

```diff
@@ -0,0 +1,30 @@
+using Offices.Application.Services;
+using Offices.Data.Repositories;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+// Adding services to container
+builder.Services.AddScoped<IOfficeService, OfficeService>();
```

Comment:
Move this to the separate extension method for ServicesProvider in application layer where you will have all dependencies in the same place for the specific project. And call this extension in program.cs

Reply:
Fixed